uplc-builtins.md

```diff
@@ -0,0 +1,28 @@
+# UPLC Builtins
+
+```k
+require "uplc-polymorphic-builtins.md"
+require "uplc-integer-builtins.md"
+require "uplc-bytestring-builtins.md"
+require "uplc-crypto-builtins.md"
+require "uplc-string-builtins.md"
+require "uplc-data-builtins.md"
+
+module UPLC-BUILTINS
+  imports UPLC-POLYMORPHIC-BUILTINS
+  imports UPLC-INTEGER-BUILTINS
+  imports UPLC-BYTESTRING-BUILTINS
+  imports UPLC-CRYPTO-BUILTINS
+  imports UPLC-STRING-BUILTINS
+  imports UPLC-DATA-BUILTINS
+
+  rule <k> (builtin BN) => < builtin BN .List #numArgs(BN) > ... </k>
+       <env> _ => .Map </env>
+    requires notBool isPolyBuiltinName(BN)
+     andBool notBool BN ==K chooseData
```

Comment:
I think I would move `chooseData` to `PolyBuiltinName` and simplify this even further.

Other than that everything looks great to me!

Reply:
I tried this initially, it broke the test [`tests/textual/new-syntax/chooseDataByteString-part5.uplc`](https://github.com/runtimeverification/plutus-core-semantics/blob/master/tests/textual/new-syntax/chooseDataByteString-part5.uplc) because the way `chooseData` gets discharged changes. Do you think it's ok to make the change and update the tests, or would that break conformance?

Reply:
Discharge behaves differently on `PolyBuiltinName`. The output of the test will have a `force` which isn't in the expected output.

https://github.com/runtimeverification/plutus-core-semantics/blob/87cb7eacb421cd60b0eaffc9e0d1580472054f8f/uplc-discharge.md?plain=1#L26-L30